File: 1 semestr/lab6/lab6_1cpp/lab6_1/lab6_1.cpp
#include <iostream>
#include <cmath>
using namespace std;

void tabul() {
    double b = 1.5, x = 0.1, y;
    while ((x >= 0.1) and (x <= 1.1)) {
        if (b * x < 1) {
            y = (b * x - log10(b * x));
        }
        else if (b * x > 1) {
            y = (b * x + log10(b * x));
        }
        else {
            y = 1;
        }
        cout << "x = " << x << " y = " << round(y * 1000) / 1000 << endl;
        x = x + 0.1;
    }
}

int main() {
    tabul();
}

File: 1 semestr/lab6/lab6_1.py
import math
def tabul():
    b = 1.5
    x = .1
    while .1<=x<=1:
        if b * x < 1:
            y = round((b*x - math.log10(b*x)),3)
        elif b * x > 1:
            y = round((b*x + math.log10(b*x)),3)
        else:
            y = 1
        print ('x = {}  y = {}'.format(x,y))
        x = round((x+.1),1)
tabul()